rpcserver.go

```diff
@@ -44,9 +44,9 @@ import (
 
 // API version constants
 const (
-	jsonrpcSemverString = "1.1.0"
-	jsonrpcSemverMajor  = 1
-	jsonrpcSemverMinor  = 1
+	jsonrpcSemverString = "2.0.0"
```

Comment:
ACK. Changed to 1.2.0, since this is intentionally written to _not_ be a breaking change. Because the `rescanblocks` and `loadtxfilter` commands are submitted as a single PR, I only upped the version by one minor version instead of two.